core/pom.xml

```diff
@@ -127,6 +127,16 @@
     </dependency>
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-proxy</artifactId>
```

Comment:
These new dependencies need to be added to the `copy-dependencies` invocation later in this file.